sdks/python/apache_beam/__init__.py

```diff
@@ -74,16 +74,22 @@
 
 from __future__ import absolute_import
 
-import sys
+import logging
 import os
+import sys
 
 
-if not ((sys.version_info[0] == 2 and sys.version_info[1] == 7) or
-        (sys.version_info[0] == 3 and
-         os.environ.get('BEAM_EXPERIMENTAL_PY3', False))):
+if sys.version_info[0] == 3:
+  logging.warning(
+      'Running the Apache Beam SDK on Python 3 is not yet fully supported. '
```

Comment:
Should we encourage users to report issues to the mailing list or BEAM-1251? Or would that be is that too noisy?

Reply:
We can modify this message as we collect more known issues (maybe document this on the website) and get closer to the release.  We will track this through the validation bug https://issues.apache.org/jira/browse/BEAM-6583.